java-script/package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "creditcoin-ts",
+    "version": "0.0.1",
+    "description": "Polkadot JS examples and integration tests for Creditcoin Substrate",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/gluwa/creditcoin-substrate.git"
+    },
+    "license": "Unlicense",
+    "bugs": {
+        "url": "https://github.com/gluwa/creditcoin-substrate/issues"
+    },
+    "homepage": "https://github.com/gluwa/creditcoin-substrate#readme",
+    "packageManager": "yarn@3.2.0",
+    "scripts": {
+        "start": "npx ts-node -O '{\"module\":\"commonjs\"}' ./src/index.ts",
+        "lint": "tsc --noEmit --pretty",
```

Comment:
This doesn't run. Gives the following error:

```
yarn run v1.22.17
$ tsc --noEmit --pretty
error TS5053: Option 'emitDeclarationOnly' cannot be specified with option 'noEmit'.
```

Reply:
Is already fixed in #125